### PR TITLE
add support for chrome 99+

### PR DESCRIPTION
**Pull request by crossjs**
fix: #69

### CODE REVIEW

**Comment by brandonbloom** [comment]
Just wanted to confirm: I build/installed this change locally and it does indeed fix the issue. Thanks!

**Comment by HoffmannP** [comment]
Works for me two, please merge

**Comment by Little-Star-Star** [comment]
It works good for me in v99 of chrome.

**Comment by c19** [comment]
the pr version works for me in v100.0.4896.60 of chrome, macos

**Comment by jefflafay** [comment]
This works on my macos with chrome 100.0.4896.88  (x86_64). The production chrome plugin is still broken without this PR.

**Comment by ianyxtan** [comment]
This fix didn't work for me. Used firefox at the mean time.

**Comment by GM-David** [comment]
Nice 👍

**Comment by mgrubinger** [comment]
Are there any plans to merge this PR?

**Comment by bershanskiy** [comment]
> Are there any plans to merge this PR?

Not that I know of. This repository appears to be abandoned (last commit in June 2021). If you like to, you can build extension locally or even just open the `ChromeData` folder and patch it manually.

**Review by rocketinventor — APPROVED** [review]
Fixes the issue! Awesome!

**Comment by bershanskiy** [comment]
Relevant Chromium bug: https://bugs.chromium.org/p/chromium/issues/detail?id=1364293

File: src/background.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const toolsPorts = new Map()
 
 chrome.runtime.onConnect.addListener(port => {
-  if (port.sender.url == chrome.runtime.getURL('/devtools/panel.html')) {
+  if (port.sender.url.replace('//devtools', '/devtools') == chrome.runtime.getURL('/devtools/panel.html')) {
```
Comment (Saibamen) [inline src/background.js:4]:
```suggestion
  if (port.sender.url.replaceAll('//', '/') == chrome.runtime.getURL('/devtools/panel.html')) {
```

And what do you think about adding `toLowerCase()`?

**Review by Saibamen — CHANGES_REQUESTED** [review]

**Comment by bershanskiy** [comment]
The Chromium DevTools bug (linked above) was fixed and fix was already rolled into Chromium. I tested on Chromium Version 108.0.5358.0 (Developer Build) (64-bit) and `sender.url` was fine. Most likely, Chromium Canary was already fixed too. We can expect Chromium 108 to release with Svelte DevTools working again.

**Comment by bershanskiy** [comment]
I just checked on Google Chrome Beta 108 and Svelte Devtools worked fine. If you still would like to use this extension, you can just install Chrome Beta or wait until November 29 for Chrome 108 to be released officially.

Test details:
 - Svelte Devtools 1.3.0 installed from CWS
 - Google Chrome Beta 108.0.5359.22
 - macOS Montrey 12.6 on MacBook Pro (Retina, 13-inch, Early 2015)

**Pull request closed without merging**